man/systemd.exec.xml

```diff
@@ -968,6 +968,31 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>BindPaths=</varname></term>
+        <term><varname>BindReadOnlyPaths=</varname></term>
```

Comment:
I'd tried to follow the existing nomenclature. Which is hard, since nspawn and service files deviate there. But
there's already `ReadOnlyPaths=` for service files, which puts "ReadOnly" before "Paths". And then there's `BindReadOnly`= and `OverlayReadOnly=` which put `ReadOnly` after the kind of mount, i.e. "Bind". Hence I came to the conclusion to name this `BindReadOnlyPaths=` as the combination of both naming schemes.

(And we probably should rename the .nspawn options, so that they also include a `Paths` as last word of the settings name, but that's a different story.)